ngraph/test/backend/experimental_detectron_generate_proposals.in.cpp

```diff
@@ -0,0 +1,127 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+// clang-format off
+#ifdef ${BACKEND_NAME}_FLOAT_TOLERANCE_BITS
+#define DEFAULT_FLOAT_TOLERANCE_BITS ${BACKEND_NAME}_FLOAT_TOLERANCE_BITS
+#endif
+
+#ifdef ${BACKEND_NAME}_DOUBLE_TOLERANCE_BITS
+#define DEFAULT_DOUBLE_TOLERANCE_BITS ${BACKEND_NAME}_DOUBLE_TOLERANCE_BITS
+#endif
+// clang-format on
+#include <numeric>
+#include "gtest/gtest.h"
+#include "runtime/backend.hpp"
+#include "ngraph/runtime/tensor.hpp"
+#include "ngraph/ngraph.hpp"
+#include "util/all_close.hpp"
+#include "util/all_close_f.hpp"
+#include "util/known_element_types.hpp"
+#include "util/ndarray.hpp"
+#include "util/random.hpp"
+#include "util/test_control.hpp"
+#include "util/test_tools.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+using Attrs = op::v6::ExperimentalDetectronGenerateProposalsSingleImage::Attributes;
+using ExperimentalGP = op::v6::ExperimentalDetectronGenerateProposalsSingleImage;
+
+static string s_manifest = "${MANIFEST}";
+
+NGRAPH_TEST(${BACKEND_NAME}, experimental_detectron_generate_proposals_eval)
+{
+    Attrs attrs;
+    attrs.min_size = 0;
+    attrs.nms_threshold = 0.699999988079071;
+    attrs.post_nms_count = 6;
+    attrs.pre_nms_count = 1000;
+
+    auto im_info = std::make_shared<op::Parameter>(element::f32, Shape{3});
+    auto anchors = std::make_shared<op::Parameter>(element::f32, Shape{36, 4});
+    auto deltas = std::make_shared<op::Parameter>(element::f32, Shape{12, 2, 6});
+    auto scores = std::make_shared<op::Parameter>(element::f32, Shape{3, 2, 6});
+
+    auto proposals = std::make_shared<ExperimentalGP>(im_info, anchors, deltas, scores, attrs);
+
+    auto f0 = make_shared<Function>(OutputVector{proposals->output(0)},
+                                    ParameterVector{im_info, anchors, deltas, scores});
+    auto f1 = make_shared<Function>(OutputVector{proposals->output(1)},
+                                    ParameterVector{im_info, anchors, deltas, scores});
+
+    auto backend = runtime::Backend::create("${BACKEND_NAME}");
+
+    std::vector<float> im_info_data = {1.0f, 1.0f, 1.0f};
+    std::vector<float> anchors_data = {
```

Comment:
It is not a meaningful test if it outputs all the same data, could you add test with better data difference.